admin/tim_limitless_consts.php

```diff
@@ -5,10 +5,11 @@
 const TIM_LIMITLESS_VIEWKEY = 'tim_limitless_viewkey';
 const TIM_LIMITLESS_ENABLED = 'tim_limitless_enabled';
 const TIM_LIMITLESS_POST_HASH = 'tim_limitless_post_hash';
-const TIM_LIMITLESS_DOMAIN = 'mediamart.tv';
-const TIM_LIMITLESS_POST_HASH_URL = 'http://localhost:8080/wordpress/posthash';
-const TIM_LIMITLESS_KEYS_URL = 'http://localhost:8082/api/wordpress_campaign?site_domain=';
-const TIM_LIMITLESS_AUDIO_URL = 'http://localhost:8080/process';
+const TIM_LIMITLESS_STARTUP = "https://trinitymedia.ai/sas/player/amazon_plugin/startup.php";
+const TIM_LIMITLESS_AUDIO_SILENCE = "https://vd.trinitymedia.ai/loading.mp3";
+const TIM_LIMITLESS_POST_HASH_URL = 'https://audio.thetimmedia.com/wordpress/posthash';
```

Comment:
`audio.thetimmedia.com` ==> `audio.trinityaudio.ai`
`r2v2.memeglobal.com` ==> `signup.trinityaudio.ai`  

@royasoolin @borisjaskerovich  open for suggestions here
about  `trinitymedia.ai` - keep it for now